system/netdata-openrc.in

```diff
@@ -25,7 +25,7 @@ extra_started_commands="getconf"
 pidfile="/run/netdata/netdata.pid"
 command="${NETDATA_INSTALL_PATH}/usr/sbin/netdata"
 command_background="yes"
-command_args="${NETDATA_EXTRA_ARGS}"
+command_args="-pidfile ${pidfile} ${NETDATA_EXTRA_ARGS}"
```

Comment:
no... in openrc you do not pass this pidfile to application.

modify: pidfile="/var/run/netdata.pid"
remove: -pidfile ${pidfile}
add: start_stop_daemon_args="--user netdata:netdata"

add:
```
start_pre() {
	mkdir -p /var/cache/netdata
	chown netdata:netdata /var/cache/netdata
}
```